src/promptflow-devkit/promptflow/_cli/_pf/_service.py

```diff
@@ -284,10 +308,11 @@ def show_service():
         log_file = HOME_PROMPT_FLOW_DIR / PF_SERVICE_LOG_FILE
     else:
         log_file = get_current_env_pfs_file(PF_SERVICE_LOG_FILE)
+    extra_info = {"log_file": log_file.as_posix(), "local_promptflow_version": get_pfs_version()}
     if status:
-        status.update({"log_file": log_file.as_posix()})
+        status.update(extra_info)
         print(status)
         return
     else:
-        logger.warning(f"Promptflow service is not started. log_file: {log_file.as_posix()}")
+        logger.warning(f"Promptflow service is not started. {extra_info}")
```

Comment:
what's the final message user will see?

Reply:
![image](https://github.com/microsoft/promptflow/assets/26239730/12d49464-0ddc-4651-9b8f-399b9a470956)